src/js/index.js

```diff
@@ -0,0 +1,248 @@
+const newsAPIKey = "8483a39eb2d649bb80b09b3c36cb30df";
+const newsDisplay = document.querySelector("#app__content-body--news");
+const readLater = localStorage.getItem("readLater")
+  ? JSON.parse(localStorage.getItem("readLater"))
+  : [];
+
+const liveNewsButton = document.querySelector("#app__header-nav--live");
+const readLaterButton = document.querySelector("#app__header-nav--readlater");
+const searchNavButton = document.querySelector("#app__header-nav--search");
+const mainNav = document.querySelector("#app__header-nav--wrapper");
+const contentBody = document.querySelector("#app__content-body");
+const mainNavItems = document.querySelectorAll(
+  "#app__header-nav--wrapper ul > li"
+);
+const navOptions = document.querySelector("#app__content-nav-options");
+
+const showImagesButton = document.querySelector("#show-images");
+const searchButton = document.querySelector("#app__content-nav-options");
+let liveNewsCollection = "";
+let searchDataCollection = "";
+let totalPages = 1;
+let page = 1;
+let maxItemPerPage = 5;
+
+function createRequest(
+  service,
+  searchTerm,
+  sortBy = "",
+  language = "",
+  page = 1
+) {
+  return service === "live"
+    ? `https://newsapi.org/v2/top-headlines?language=en&apiKey=${newsAPIKey}&page=${page}&pageSize=${maxItemPerPage}`
+    : `https://newsapi.org/v2/everything?q=${searchTerm}&language=${language}&sortBy=${sortBy}&apiKey=${newsAPIKey}&page=${page}&pageSize=${maxItemPerPage}`;
+}
+
+function displayNews(news) {
+  const articles = [];
+  news.forEach(article => {
+    articles.push(generateArticle(article));
+  });
+  newsDisplay.innerHTML = articles.join("");
+}
+
+function generateArticle(article) {
+  return `
+  <article class="article">
+        <a target="_blank" class="article__link" href="${article.url}">
+        <header class="article__title">${article.title}</header>
+            <figure class="article__image"
+                    style="background-image:url('${article.urlToImage}')">
+            </figure>
+            <span id="article__date" class="article__date">${new Date(
+              article.publishedAt
+            ).toGMTString()}</span>
+            <span class="article__author">${article.author}</span>
+            <p class="article__description">${article.description}</p>
+        </a>
+        <button type="text" data-title="${
+          article.title
+        }" class="article__readlater btn">Read later</button>
+    </article>
+    `;
+}
+
+function toggleImages() {
+  const localStor = localStorage.getItem("showImages");
+  const images = document.querySelectorAll(".article__image");
+  const checkbox = document.querySelector("#show-images");
+  if (localStor === "show") {
+    checkbox.checked = true;
+    images.forEach(function(img) {
+      img.style.display = "block";
```

Comment:
it would be better to set a class whoch controls if image is displayed and then toggle the class rather than manipulating style directly. On larger projects this class could be re-used across other components